desk/app/chat.hoon

```diff
@@ -1345,7 +1357,9 @@
       ::
           %del
         =?  cu-core  ?=(^ had)
-          (cu-activity [%delete p.diff.delta time.u.had] *story:d |)
+          =*  content  content.writ.u.had
```

Comment:
If we ever add edits to chat this might start failing, but for now we should consider ourselves lucky...

Reply:
ha